source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -221,6 +221,7 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
 | `temporarily_unavailable`   | If you're only making an authentication request (as opposed to requesting both authentication and identity), this error code means the GOV.UK One Login authentication server is temporarily unavailable, which might be caused by temporary overloading or planned maintenance. <br>Make your request again in a few minutes. <br>  <br> If you're making an identity request and you get this error, it means the identity proving and verification does not currently have capacity for this request.                                                                                                                                                                                                                                                                                       |
 | `access_denied`             | GOV.UK One Login returns this error in 3 potential scenarios.<br><br>The first scenario is that the identity evidence your user provided has a lower score than the identity confidence specified in your request. As a result, GOV.UK One Login could not return the medium level of identity confidence (`P2`) and instead returned a lower level of identity confidence.<br><br> If you’re using return codes, you will not receive this error. Instead, GOV.UK One Login returns an array of single letter `returnCode` values through the `userinfo` endpoint. Find more information on [understanding the return codes claim](/integrate-with-integration-environment/prove-users-identity/#understand-your-user-s-return-code-claim).<br><br>The second scenario is that your user selected **Cancel** during the identity verification process.<br><br>The third scenario is that the session in the user’s browser is unavailable. This can happen when your user’s cookies have been lost or your user changed browsers during the identity verification process. In this scenario, you should ask your user to log in again or restart the identity verification process. |
+| `login_required`            | If you make a re-authentication request and your user is unable to authenticate themselves.<br>Reasons a user may not be able to able to authenticate themselves when re-authentication is requested are:<ul><li>the user has entered a different email address to the one supplied by you</li><li>the user has entered the wrong password too many times</li><li>the user has entered the wrong OTP code too many times</li></ul><br> |
```

Comment:
```suggestion
| `login_required`            | You have made a re-authentication request but your user is unable to authenticate themselves.<br>Reasons a user may not be able to able to authenticate themselves after you've requested re-authentication are:<ul><li>the user has entered a different email address to the one you have for them</li><li>the user has entered the wrong password too many times</li><li>the user has entered the wrong one-time password (OTP) code too many times</li></ul><br> |
```

Reply:
changed 'supplied by you' as it wasn't clear how the RP would supply this. If 'to the one you have for them' is incorrect, flag here and we can rework

Reply:
Hmm actually I believe it should really say 'the one we have for them'

Reply:
I've made that change, thanks